extensions/2.0/Khronos/KHR_materials_volume/README.md

```diff
@@ -0,0 +1,255 @@
+</figure>
+
+## Transmission and Translucency
+
+The volume extension needs one of `KHR_materials_transmission` and `KHR_materials_translucency` to allow light rays to pass through the surface into the volume. Once the volume is entered however, the simulation of absorption and subsurface scattering inside the medium will be independent of the surface properties.
+
+If the extension is combined with `KHR_materials_transmission`, the refraction occurs at the microfacets. That means that the thin microfacet BTDF is replaced by a microfacet BTDF that takes refraction into account. The roughness parameter affects both reflection and transmission.
+
+<figure style="text-align:center">
+<img src="./figures/transmissive-roughness.png"/>
+<figcaption><em>Transmissive sphere with varying roughness. From left to right: 0.0, 0.2, 0.4.</em></figcaption>
+</figure>
+
+If the extension is combined with `KHR_materials_translucency`, the translucent BTDF remains unchanged.
+
+<figure style="text-align:center">
+<img src="./figures/translucent-roughness.png"/>
+<figcaption><em>Translucent sphere with varying roughness. From left to right: 0.0, 0.2, 0.4.</em></figcaption>
+</figure>
+
+For best results, we recommend using translucency in case the medium exhibits strong subsurface scattering (small scattering distance, high subsurface color). Real-time implementations may use translucency as a cue to switch to approximations for subsurface scattering that work well for materials like skin or candle wax. For these dense materials, the visual difference between transmission and translucency is small, as the path a light travels is dominated by volume scattering. The scattering interaction at the volume boundary has only a small effect on the final result. In some cases it might even be possible to fake subsurface scattering with translucency and a thin-walled material.
+
+<figure style="text-align:center">
+<img src="./figures/transmission-translucency.png"/>
+<figcaption><em>Comparison of combining subsurface scattering with either transmission or translucency. Left: Rough transmission and subsurface scattering. Middle: Translucency and subsurface scattering. Right: Translucency without subsurface scattering using a thin-walled material. Colors are adjusted manually so that they look similar in the three configurations. This adjustment is needed in order to account for differences in distances and to minimize the impact of energy loss from the rough microfacet BTDF.</em></figcaption>
+</figure>
+
+## Implementation
+
+*This section is non-normative.*
+
+The microfacet BTDF f<sub>transmission</sub> defined in `KHR_materials_transmission` now takes refraction into account.
+
+```
+                  abs(LdotH) * abs(VdotH)                 ior_o^2 * G * D
+f_transmission = ------------------------- * ----------------------------------------
+                  abs(LdotN) * abs(VdotN)      (ior_i * VdotH + ior_o * LdotH)^2
+
+```
```

Comment:
It's not too clear while reading this that `f_transmission` must use the modified H value defined below.  Perhaps we could define the modified half vector first? 

I would also recommend that `D` be renamed to `Dt` to be consistent with KHR_material_transmission notation.  `Dt` is more clear in that D that uses the modified half vector.